osbenchmark/builder/models/cluster_flavors.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class ClusterFlavor(str, Enum):
+    SELF_MANAGED = "self_managed"
+    MANAGED = "managed"
```

Comment:
Do we need two different enums for ClusterFlavor & ClusterInfraProvider? Can you explain what's the use case supporting the need for two enums.

I was thinking of combining them into one. The `MANAGED` flavor seems too generic for me, similarly `LOCAL` provider probably doesn't add much value.